src/lib/select/select.spec.ts

```diff
@@ -19,7 +19,7 @@ import {
 } from '@angular/forms';
 import {ViewportRuler} from '../core/overlay/position/viewport-ruler';
 
-describe('MdSelect', () => {
+fdescribe('MdSelect', () => {
```

Comment:
It is, but it's not the only issue. I'll get it passing once I get some feedback on it.